plugins/newsdownloader.koplugin/main.lua

```diff
@@ -111,22 +110,24 @@ function NewsDownloader:loadConfigAndProcessFeeds()
     UIManager:forceRePaint()
     UIManager:close(info)
 
-    if not lfs.attributes(FEED_CONFIG_PATH, "mode") then
+    if not lfs.attributes(feed_config_path, "mode") then
         logger.dbg("NewsDownloader: Creating initial feed config.")
-        FFIUtil.copyFile(FFIUtil.joinPath(self.path, FEED_CONFIG_FILE),
-                         FEED_CONFIG_PATH)
+        FFIUtil.copyFile(FFIUtil.joinPath(self.path, feed_config_file),
+                         feed_config_path)
     end
-    local ok, feed_config = pcall(dofile, FEED_CONFIG_PATH)
+    local ok, feed_config = pcall(dofile, feed_config_path)
     if not ok or not feed_config then
         logger.info("NewsDownloader: Feed config not found.")
         return
     end
 
     if #feed_config <= 0 then
-        logger.info('NewsDownloader: empty feed list.', FEED_CONFIG_PATH)
+        logger.info('NewsDownloader: empty feed list.', feed_config_path)
         return
     end
 
+    local unsupported_feeds_urls = {};
```

Comment:
luacheck only warns about semicolons that cause syntax errors such as `do … end;`. It's never been an issue before. I'm not sure it's worth the extra effort just because you like to sneak in semicolons. :-P